pootle/static/css/navbar.css

```diff
@@ -262,62 +459,8 @@ html[dir="rtl"] #nav-access
         clear: both;
     }
 
-    #nav-main
+    #nav-main .js-score
```

Comment:
Note that for the same reasons we prefix selectors targeting JS with `js-`, we shouldn't be using `js-`-prefixed selectors if targeting CSS.